Assets/Scripts/Wall/WallDanger.cs

```diff
@@ -26,21 +27,21 @@ private void Start()
 
         public IEnumerator BecameDangerousCoroutine(float wallWarningInterval)
         {
-            //_spriteRenderer.sprite = warning;
             _animation.MakeWarning();
             _lighting.SetWarning();
 
             yield return new WaitForSeconds(wallWarningInterval);
 
-            //_spriteRenderer.sprite = danger;
             _animation.BecameDanger();
+
+            yield return new WaitForSeconds(waitingDangerAnimationEndingIntrval);
```

Comment:
  TODO should be added, otherwise, it's easy to forget to change if the animation length is changed.